static/app/views/organizationGroupDetails/quickTrace/configureDistributedTracing.tsx

```diff
@@ -192,7 +192,7 @@ const ExampleQuickTracePanel = styled(Panel)`
   gap: ${space(1)};
   background: none;
   padding: ${space(2)};
-  margin: ${space(2)} 0;
+  margin: ${space(2)} 0 0;
```

Comment:
What this looks like without the alert:
<img width="869" alt="Screen Shot 2022-08-17 at 2 59 45 PM" src="https://user-images.githubusercontent.com/44172267/185250848-829524d5-0e5a-4918-b036-18d32e147f85.png">